tensorflow/compiler/xla/client/lib/pooling.cc

```diff
@@ -198,15 +198,16 @@ XlaOp AvgPoolGrad(XlaOp out_backprop, absl::Span<const int64> gradients_size,
   XlaBuilder* b = out_backprop.builder();
   return b->ReportErrorOrReturn([&]() -> StatusOr<XlaOp> {
     const int num_dims = kernel_size.size();
-
-    if (gradients_size.size() != num_dims) {
+    const int gradients_size_size = gradients_size.size();
+    if (gradients_size_size != num_dims) {
       return tensorflow::errors::InvalidArgument("gradients must be ", num_dims,
                                                  "-dimensional");
     }
 
     TF_ASSIGN_OR_RETURN(Shape out_backprop_xla_shape,
                         b->GetShape(out_backprop));
-    if (out_backprop_xla_shape.dimensions().size() != num_dims) {
+    const int out_backprop_xla_shape_dimensions_size =out_backprop_xla_shape.dimensions().size();
```

Comment:
This line might be too long. Why not just use `size`?

Reply:
I default to verbose.  Just `size` could be hard to disambiguate.